viewing_party/party.py

```diff
@@ -0,0 +1,193 @@
+# Wave 1
+def create_movie(title, genre, rating):
+    """
+    This function returns a dictionary that gives information about a
+    movie's title, genre and rating when all 3 parameters are truthy. 
+    Otherwise, the function returns None.
+    """
+    new_movie = {}
+    if title and genre and rating:
+        new_movie["title"] = "Title A"
+        new_movie["genre"] = "Horror"
+        new_movie["rating"] = 3.5
+        return new_movie
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    """  
+    The function returns a dictionary that gives information about the 
+    movies that the user has watched. This function adds another 
+    dictionary, movie, to the list of dictionaries in user_data. 
+    """
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    """
+    The function returns a dictionary that gives information about a 
+    user's watchlist. This function adds another dictionary, movie, 
+    to the list of dictionaries in user_data.
+    """
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title): 
+    """
+    The function returns a dictionary that gives information about 
+    movies that the user has watched and movies that are on the 
+    user's watchlist. If the title is in the user's watchlist, 
+    the function will remove the movie from the watchlist and add 
+    it to the list of movies that the user has already watched. 
+    Otherwise, the function will simply return user_data. 
+    """
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
```

Comment:
this is a very redundant suggestion since this is only one line of code and thus works just fine, but where have we seen this before? maybe we could use `add_to_watched` to do this for us. Again, suuuper unnecessary change to make in the scheme of things, so think of this as just something to consider for next time.
```suggestion
 add_to_watched(movie)
```